projects/cdk/schematics/ng-update/constants/deprecated-functions.ts

```diff
@@ -353,7 +353,7 @@ export const DEPRECATED_FUNCTIONS: readonly TypeToRename[] = [
     },
     {
         from: 'getClosestKeyboardFocusable',
-        to: 'tuiGetClosestKeyboardFocusable',
+        to: 'tuiGetClosestFocusable',
```

Comment:
now we need more complex migration like here https://github.com/Tinkoff/taiga-ui/blob/v3.x/projects/cdk/schematics/ng-update/steps/replace-functions.ts#L106

nit: bwt why is pr not directed in 3.0 branch?